Nodejs/Product/Nodejs/Repl/NpmReplCommand.cs

```diff
@@ -137,7 +111,20 @@ class NpmReplCommand : IReplCommand {
             }
 
             var npmReplRedirector = new NpmReplRedirector(window);
-               
+            
+            if (!isGlobalCommand && !File.Exists(Path.Combine(projectDirectoryPath, "package.json"))) {
```

Comment:
On second thought, I think I'll revert this part of the change. We could try checking and only doing this  for npm commands that actually need the `package.json`, but i think it is better to let the user handle this themselves using `npm init`.